Livecheckables/coinutils.rb

```diff
@@ -0,0 +1,4 @@
+class Coinutils
+  livecheck :url   => "https://github.com/coin-or/CoinUtils.git",
+            :regex => %r{^releases\/(\d+(?:\.\d+)+)$}i
```

Comment:
```suggestion
            :regex => %r{^(?:releases/)?(\d+(?:\.\d+)+)$}i
```

As before, we can unescape forward slashes when using the `%r{}` regex literal syntax.

We could also make `releases/` optional, which would match tags like `1.2.3` and `releases/1.2.3`, in case upstream ever moves away from the `releases/` prefix. It's not explicitly necessary but it probably wouldn't hurt.